tree-sitter-verilog/build.sh

```diff
@@ -0,0 +1,11 @@
+#! /bin/bash
+
+set -e
+set -x
+
+npm config set python python2.7
+npm install
+
+$PYTHON -m pip install --isolated tree_sitter
+$PYTHON -c "from tree_sitter import Language; Language.build_library(\"$PREFIX/lib/verilog.so\", [\".\"])"
```

Comment:
Should this be something like `$PREFIX/lib/tree-sitter-verilog.so`?

Reply:
`verilog.so` might be too generic, I've changed it to what you suggested